go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/abcxyz/jvs-plugin-jira
 go 1.20
 
 require (
-	github.com/abcxyz/jvs v0.0.0-20230623161235-a02ba7d38f4b // TODO update to major version
+	github.com/abcxyz/jvs v0.0.0-20230718165522-9c429bb50dca // TODO update to major version
 	github.com/abcxyz/pkg v0.4.0
 	github.com/google/go-cmp v0.5.9
 	github.com/hashicorp/go-plugin v1.4.10
 )
 
 require (
-	cloud.google.com/go/compute v1.19.3 // indirect
```

Comment:
Sanity check, are those auto-generated due to the `github.com/abcxyz/jvs` update?

Reply:
Yes.